slides/index.html

```diff
@@ -247,20 +261,14 @@ <h1 style="color:DeepSkyBlue">Docker</h1>
         <section>
           <h3>Docker Environment</h3>
           <pre><code class="bash" data-trim data-noescape>
-$ alias de="env | grep DOCKER_"
+$ cd Downloads/mycluster
 
-<span class="fragment">$ cd Downloads/mycluster</span>
+<span class="fragment">$ source docker.env
+> . .\docker.ps1</span>
 
-<span class="fragment">$ cat README.md</span>
```

Comment:
Removed this since it doesn't add much and it was causing the code area to scroll, cutting off content